plugins/modules/nomad_token.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2023, Pedro Nascimento <apecnascimento@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: nomad_token
+author: Pedro Nascimento(@apecnascimento)
+version_added: "1.0.0"
+short_description: Manage Nomad ACL tokens
+description:
+    - Create Bootstrap token
+    - Create ACL token.
+    - Update ACL token.
+    - Delete ACL token
+requirements:
+    - python-nomad
+extends_documentation_fragment:
+    - community.general.nomad
+    - community.general.attributes
+attributes:
+    check_mode:
+        support: full
+    diff_mode:
+        support: none
+options:
+    name:
+        description:
+            - Name of acl token to create.
+        type: str
+    token_type:
+        description:
+            - The type of the token can be "client", "management" or bootstrap.
+        choices: ["client", "management", "bootstrap"]
+        type: str
+        default: "client"
+    policies:
+        description:
+            - A list of the policies assigned to the token.
+        type: list
+        elements: str
+        default: []
+    global_replicated:
+        description:
+            - indicates whether or not the token was created with the --global.
```

Comment:
```suggestion
            - Indicates whether or not the token was created with the C(--global).
```
The sentence feels incomplete, should there be `option` at the end?